lib/render/html.js

```diff
@@ -288,12 +288,15 @@ function table_row(node, entering) {
         this.cr();
     } else {
         this.tag("/tr");
+        this.cr();
```

Comment:
cmark-gfm handles whitespace slightly differently in its HTML output, so I had to change that for our tests as well so that we could use the tests from cmark-gfm (`test/gfm_extensions.txt`)